ironic/Dockerfile

```diff
@@ -5,6 +5,7 @@ RUN curl https://raw.githubusercontent.com/openstack/tripleo-repos/master/triple
 RUN yum install -y openstack-ironic-api openstack-ironic-conductor rabbitmq-server crudini iproute dnsmasq httpd qemu-img iscsi-initiator-utils
 RUN mkdir -p /var/www/html/images
 RUN curl http://tarballs.openstack.org/ironic-python-agent/tinyipa/tinyipa-stable-rocky.tar.gz | tar -C /var/www/html/images/ -xzf -
+RUN curl --insecure --compressed -L -o /var/www/html/images/redhat-coreos-maipo-47.284-openstack.qcow2 https://releases-redhat-coreos-dev.cloud.paas.upshift.redhat.com/storage/releases/maipo/47.284/redhat-coreos-maipo-47.284-openstack.qcow2.gz
```

Comment:
I think we'll have to update this (and get_rhcos_image.sh) to use the external URL which I think is https://releases-rhcos.svc.ci.openshift.org/storage/releases/maipo/47.284/redhat-coreos-maipo-47.284-openstack.qcow2.gz

Perhaps we can update this later so we sed the URL and release from the scripts, so we only have the location/version in one place.